test/integration/content-explorer/ContentExplorer.int.test.js

```diff
@@ -220,15 +218,49 @@ describe('ContentExplorer', () => {
                 .find(helpers.getSelector(gridViewClass))
                 .should('have.length', 0);
         });
+    });
 
-        it('Should switch to grid view', () => {
+    describe('Grid View', () => {
+        beforeEach(() => {
+            helpers.load({ features: gridViewOn });
             helpers.getViewModeChangeButton().click();
+        });
+
+        it('Should switch to grid view', () => {
             cy.getByTestId('content-explorer')
                 .find(helpers.getSelector(listViewClass))
                 .should('have.length', 0);
             cy.getByTestId('content-explorer')
                 .find(helpers.getSelector(gridViewClass))
                 .should('have.length', 1);
         });
+
+        it('Should open and close share modal', () => {
+            cy.getByAriaLabel(localize('be.share')).should('have.length', 0);
+            cy.contains(localize('be.share')).click();
+            cy.getByAriaLabel(localize('be.share')).should('have.length', 1);
+            helpers.getCloseButton().click();
+            cy.getByAriaLabel(localize('be.share')).should('have.length', 0);
+        });
+
+        it('Should open and close rename modal', () => {
+            cy.getByAriaLabel(localize('be.rename')).should('have.length', 0);
```

Comment:
It's not super clear to me that we're trying to test a modal rather than a button, since the names on everything are so similar. Can we select the modal or modal header itself to make it more clear?